tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": "src",
     "target": "ES6",
     "module": "ESNext",
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Why?

Reply:
Here is a good explination [react vs. react-jsx](https://stackoverflow.com/a/73518971).
Basicly, it's the modern way of writing React. It will break compatibility with existing React 16 Projects, but i doubt that this would be a issue, because:

- React 17 predates Directus 9 and this Repo itself and is now only used in legacy Projects. So i doubt they would integrade a new nodeJS Backend in such a software.
- Upgrading from React 16 to React 17 is relativly easy and requires few modifications to the code.
- It aligns our project with the modern way of React. Improving performance and making it more consistent with other React packages.

Reply:
Sorry for being of few words, I know the difference, I was meaning: here we drop support for React 16 right? Improvements could be neglegible for such small libraries. Are we sure we want to?

Reply:
Yes, based on the above-stated reasons and the download count of this repository, I concluded that probably nobody uses it with React 16. However, you are right; we could close this for now.

I would, though, add this, alongside a couple of other breaking changes(remove Components from useDirectus, move the directus image transformation props to a single `object | key` prop, ...) when I start working on the Directus SDK v11 support, as the SDK version basically breaks all current code.

Could you enable Discussions for this repository so we could talk about the way forward (new package release, handling changes for Directus SDK v11, ...) in a more accessible and persistent manner than in PRs?

Reply:
Discussions enabled. Can I just merge this without the chage to react-jsx?

Reply:
Thanks.

Sadly the react-jsx setting in the tsconfig file is required for the new style of imports.